okonf/facts/files.py

```diff
@@ -114,6 +114,23 @@ async def enforce(self, host: Executor):
         return True
 
 
+class FileExecutable(Fact):
+    """Ensure that a file is executable"""
+
+    def __init__(self, remote_path):
+        self.remote_path = remote_path
+
+    async def enquire(self, host: Executor) -> bool:
+        command = "test -x {} && echo True || echo False".format(self.remote_path)
```

Comment:
This only tests if the file is executable by the current user, which is not consistent with making it executable by all users using `chmod a+x`.